apps/meteor/app/lib/server/functions/getRoomsWithSingleOwner.ts

```diff
@@ -64,11 +64,11 @@ export async function getSubscribedRoomsForUserWithDetails(
 				}
 
 				// If there's no subscriber available to be the new owner and it's not a public room, we can remove it.
-				if (!roomData.shouldChangeOwner && roomData.t !== 'c') {
+				if (!roomData.shouldChangeOwner) {
```

Comment:
```suggestion
				if (!roomData.shouldChangeOwner && roomData.t !== 'c') {
```
Changing this would be a change in behavior, out of the scope of this PR.

Reply:
This will be chenged in another PR